test/test_manifest.py

```diff
@@ -109,3 +110,28 @@ def test_manifest_local_checks_containers_storage_works(tmp_path, build_containe
             f'--entrypoint=["/usr/bin/bootc-image-builder", "manifest", "--local", "localhost/{container_tag}"]',
             build_container,
         ], check=True, encoding="utf8")
+
+
+@pytest.mark.skipif(platform.uname().machine != "x86_64", reason="cross build test only runs on x86")
+def test_manifest_cross_arch_check(tmp_path, build_container):
+    cntf_path = tmp_path / "Containerfile"
+    cntf_path.write_text(textwrap.dedent("""\n
+    # build for x86_64 only
+    FROM scratch
+    """), encoding="utf8")
+
+    # XXX: check go arch names vs container arch names (amd64 vs x86_64)
```

Comment:
Go and OCI arch names are the same actually.  x86_64/aarch64 are I think the GNU arch names and typically used by Linux and at least rpm.

Reply:
Thanks, yes - it kernel names vs go/oci names. I removed this again, we use the type `arch.Architeture` (mostly) consistently and that will parse both names so it's fine. It was mostly a reminder for myself to double check (which I did now).